insights/config/specs.py

```diff
@@ -391,7 +391,8 @@
     "teamdctl_state_dump"       : CommandSpec("/usr/bin/teamdctl {iface} state dump", iface=r"team\S+"),
     "tomcat_web.xml"            : First([PatternSpec(r"etc/tomcat.*/web\.xml"),
                                     PatternSpec(r"conf/tomcat/tomcat.*/web\.xml")]),
-    "tomcat_virtual_dir_context": CommandSpec("/bin/grep -R --include '*.xml' 'VirtualDirContext' /usr/share/tomcat*"),
+    "tomcat_vdc_fallback"       : CommandSpec("/usr/bin/find /usr/share -maxdepth 1 -name 'tomcat*' -exec /bin/grep -R -s 'VirtualDirContext' --include '*.xml' '{}' +"),
+    "tomcat_vdc_targeted"       : CommandSpec("/bin/grep -R -s 'VirtualDirContext' --include '*.xml' {catalina}"),
```

Comment:
You'll need to add `, catalina=r"\S+"` or a modified version to the end of this in order for it to work with the precommand.

Reply:
Does it only check if it is there? I could not find the code which handles that. I am asking because I expect either `/some/path` or `/some/path /some/other/path` as a result.